packages/persist/lib/server.ts

```diff
@@ -26,6 +27,8 @@ server.use((req: Request, res: Response, next: NextFunction) => {
     }
 });
 
+server.use('/environment/:environmentId/*', authMiddleware);
```

Comment:
zod middleware doesn't play well with other middlewares. I tried `server.post(..., [authMiddleware, zodMiddleware], ...) but got typing errors. 
Therefore using `use()` with a wildcard. Let me know if you are aware of a better solution